core/controllers/topic_editor_test.py

```diff
@@ -133,18 +220,20 @@ def test_handler_updates_story_summary_dicts(self) -> None:
             topic_id, self.admin_id, name='New name',
             abbreviated_name='topic-two', url_fragment='topic-two',
             description='New description',
-            canonical_story_ids=[canonical_story_id],
+            canonical_story_ids=[canonical_story_id, canonical_story_id_2,
```

Comment:
nit: rename `canonical_story_id` to `canonical_story_id_1` for consistency

Reply:
Done